### ISSUE TITLE
ExplorerPatcher not working on Windows 11 "Enterprise" (continuation of Issue #2486)

### ISSUE THREAD

**Issue by DeepCoreII** [valinet/ExplorerPatcher#2494]
Hello again - I am unable to communicate this information to Issue #2486 - so I have created this ticket.

I asked around yesterday to see if anything other than the Microsoft Patches happened to our workstations this week and was told "All workstations were switched over from Windows 11 Professional to Windows 11 Enterprise Monday night via a simple registry change".

Were any of the "9ish PCs of mine and my friends so far" include Windows 11 "Enterprise"?  Most individuals are not running the "Enterprise" version of Windows and due to the timing I'm pretty confident this is why the existing version (22621.1555.55.1) stopped working and why the latest release (22621.2506.60.1) will not install.

I have attached a screenshot of the "product info" registry for your review
![2023_11_22 Cannot install EP ver 22621 2506 60 1 on Win11 Enterprise](https://github.com/valinet/ExplorerPatcher/assets/73230748/27fede3d-ab3e-4f25-b186-317f3ec9a4ff)
.

Thank you again for your time.  I am on vacation for the next 4 days, but will return on Monday.

**Comment by pyrates999** [comment]
I don't think you can change it to enterprise version that way.  You'd have to input the key for enterprise, and then windows would change over to that edition.

**Comment by Antonomasia3** [comment]
for reference, it worked for me
![image](https://github.com/valinet/ExplorerPatcher/assets/89201774/ef0dc641-8af9-41f7-9fcc-c782b2b3a781)

maybe try to restore that registry back to what it was, and change edition using what pyrates said (?)
my Windows edition is Enterprise since clean install, never changed it before

**Comment by DeepCoreII** [comment]
[pyrates999] I don't think you can change it to enterprise version that way. You'd have to input the key for enterprise, and then windows would change over to that edition.

Good thinking, but I confirmed with our IT staff that they made the switch at the direction of Microsoft Support.  Obviously, they are not telling us what exactly they had to change only that "the end-users should see no difference".

**Comment by DeepCoreII** [comment]
[Antonomasia3] for reference, it worked for me (look at the "Tentang Windows" window on the top-left)
maybe try to restore that registry back to what it was, and change edition using what pyrates said (?) my Windows edition is Enterprise since clean install, never changed it before

Thank you for the idea, but that is not an option for me.  All I know is that it worked Monday (11/20/23) when I signed off work and it no longer worked the next morning.  I can Search for ExplorerPatcher Properties and look at all the settings, but I can no longer even pull up the Properties when right-clicking the Taskbar.

My screenshot shows a "before" (when ExplorerPatcher worked) and "after" pictures (when it stopped).
![2023_11_27 Explorer Patcher Properties](https://github.com/valinet/ExplorerPatcher/assets/73230748/0fd615a6-e80c-4fe3-8af5-a38ed5e173c4)
![QuickLaunch Taskbar Toolbars (Before   After)](https://github.com/valinet/ExplorerPatcher/assets/73230748/289fc60d-4225-4771-af2b-c9103e50537b)

**Comment by Jazzzed** [comment]
I do agree that your issue is likely the fact that a very old version of EP was installed when Windows updated, and now it just won’t run anymore. I don’t think the problem is directly related to the Enterprise switchover though. If you could figure out how to uninstall the old version of EP, and then reinstall the latest, you’d probably be fine. But my theory is that your IT dept has locked down the ability to do installs / uninstalls. Thats based on the screenshot in your other thread. Plus isn’t that why someone would install the Enterprise edition in the first place? Control?

**Comment by pyrates999** [comment]
If your IT department has locked down installing software, you need to ask them to uninstall EP for you and then you can't use it anymore.  I wouldn't use it on any computer used for work.
